pulsar-client-cpp/include/pulsar/c/producer_configuration.h

```diff
@@ -63,8 +63,17 @@ typedef enum {
     pulsar_AutoPublish = -4,
 } pulsar_schema_type;
 
+typedef enum {
+    // This is the default option to fail send if crypto operation fails
+    pulsar_producerFail,
+    // Ignore crypto failure and proceed with sending unencrypted messages
+    pulsar_producerSend
```

Comment:
The naming style should be consistent with other enum values, i.e. change `pulsar_producerFail` to `pulsar_ProducerFail`.